README.md

```diff
@@ -1,7 +1,39 @@
-# Auth-A-Rama Centra. 
+# Auth-A-Rama Centra.
 
 A stand-alone authorization API created with Node, Express, Mongoose, MondoDB, JWT, and React
 
+# Installation & Configuration
+
+Steps to install all the dependencies and bring this api up using docker
+
+## Install Dependencies
+
+1. Install all the npm dependencies by running npm install in the `/api` and `/frontend` directories
+
+## Configs
+
+1. there is an `.env` file in the root of `/docker/production`. A similar or the same file should exist in the root of `/docker/aarc_dev`. Copy and paste `/docker/production/.env` into `/docker/aarc_dev`.
+
```

Comment:
Possibly we should consider also adding this extra .env file. What to you think?

Reply:
Yes, I can throw that in.  I'm going to include in a different branch though, since its not documentation related and then I'll update this one accordingly

Reply:
I made the changes, but we need to merge in PR#13 for it to be correct